joomla/joomla-cms
# [4.1]Recent Articles admin module - workflows

## Purpose
If you are using workflows then it is useful to know what stage an article is at. This PR adds the display of the stage to the administrator module "recently added articles". The column is only displayed when workflows is enabled

### Test Setup Instructions

- Enable workflows in the content options
- Create several articles and set them at different stages
- check the module on the home dashboard and the content dashboard

### Before this PR
No visible means to see the stage of all the articles
![image](https://user-images.githubusercontent.com/1296369/166973396-cc5f5460-fee0-4e5c-98b3-f0e93824e58b.png)

### After this PR
The stage is now displayed if workflows is enabled
![image](https://user-images.githubusercontent.com/1296369/166973318-bf96fb3c-766e-4f00-9626-ae2d2fab840e.png)
